Sources/Sentry/SentrySDK.m

```diff
@@ -83,6 +83,28 @@ + (void)startWithOptionsObject:(SentryOptions *)options
     [SentrySDK installIntegrations];
 }
 
++ (void)startWithDsn:(NSString *)dsn
+{
+    [self startWithDsn:dsn withConfigureOptionsBlock:^(SentryOptions *options) {}];
+}
+
++ (void)startWithDsn:(NSString *)dsn
```

Comment:
H: why don't they use `startWithOptionsObject`? is there a technical reason?

Reply:
That is a good point. I think the reason is that the init of the `SentryOptions` is not the most user friendly. You have to pass in a dictionary and then afterwards you can modify the options. I think this is the root cause for the confusion of the users. 
Then I would suggest we add another possibility to init the `SentryOptions` and we change the docs so users don't init the `SentryOptions` with a dictionary.

Reply:
but why? why cant they call `start(options)` directly? why do they need a dictionary? 

```
let options = Options()
SentrySDK.start(options: options)
```

isn't this already possible? or am I missing something?

Reply:
Nope you have to do 
```
do {
    let options = try Options(dict: ["dsn": "DSN"])
    options.debug = true
    SentrySDK.start(options: options)
} catch {
    // ...
}
```

Reply:
We could also change it so you can do this:
```
do {
    let options = try Options(dsn: "DSN")
    options.debug = true
    SentrySDK.start(options: options)
} catch {
    // ...
}
```

Reply:
the 2nd option would be the unified API and the way to go IMO

Reply:
ideally it would be a block and not passing the instance. Let the SDK initalize options and pass the reference to the block

Reply:
yep just what @bruno-garcia  said, on Android is a block (interface too), but we require a Context so.

`SentryAndroid.init(context, block {})`